src/components/settings/ARDownloadListSettings.js

```diff
@@ -50,23 +46,32 @@ const renderItem = ({ data, index, item, setData }) => (
 );
 
 export const ARDownloadListSettings = () => {
-  const [data, setData] = useState(dummyData);
-  const [loading, setLoading] = useState(false);
+  const { data: staticData, loading, refetch } = useStaticContent({
+    name: 'arDownloadableDataList',
+    type: 'json'
+  });
+
+  const [data, setData] = useState([]);
+  const [isLoading, setIsLoading] = useState(loading);
```

Comment:
do we need the own state if `useStaticContent` is giving us such a variable?

Reply:
We need this. The JSON on the server only contains default data. We need to update this JSON later on the device.